test/unit/spec/narrativeLoginSpec.js

```diff
@@ -23,7 +25,7 @@ define(['jquery', 'narrativeLogin', 'narrativeConfig', 'narrativeMocks'], (
             $(document).off();
             jasmine.Ajax.uninstall();
             Mocks.clearAuthToken();
-            document.body.innerHTML = '';
+            $node.destroy();
```

Comment:
use `$node` instead of the document to avoid accidental document test cross-contamination